src/Aquifer.Jobs/local.settings.example.json

```diff
@@ -1,32 +1,19 @@
 {
+    "IsDevelopment": true,
```

Comment:
Can this move to appsettings.json instead?

Reply:
When testing, I could not get it to have any effect using it from appsettings. If we find that it does work in appsettings, I'm happy to move it

Reply:
Does it need to move back into Values, then?  https://github.com/BiblioNexusStudio/aquifer-server/pull/523/files#diff-13c6ac85989c85e36799e1fed24dba5f1e9d5ef74df72bc0a3c976396d0acaebR5 requires it to exist in the settings.